### DISCUSSION TITLE
Project management

### DISCUSSION THREAD

**Discussion by hydroper** [bevyengine/bevy#9043] [Q&A]
I'm new here and just interested on Bevy and I'm not requesting an editor, but the Godot Engine editor does the following things for you:

- It allows you to export the project to a given platform. It will allow you to quickly configure the build process and the packagement.
- Allows scaffolding new projects (what the [bevy_game_template](https://github.com/NiklasEi/bevy_game_template) does I think, but I think it could be officialized)
- It associates configuration to assets like SVGs, PNGs and fonts, what are called `.import` files (similiar to `some-svg.svg.import`). A directory can be ignored with `.gdignore`, thus causing the editor to not associate this configuration with these files in that directory recursively.

A standard command for managing Bevy projects could be interesting I think.

**Comment by dmyyy** [comment]
Might be good to mention (or see if it's mentioned) here: https://github.com/bevyengine/bevy/discussions/5043

**Comment by mockersf** [comment]
> * Allows scaffolding new projects (what the [bevy_game_template](https://github.com/NiklasEi/bevy_game_template) does I think, but I think it could be officialized)

The official version is [bevy_github_ci_template](https://github.com/bevyengine/bevy_github_ci_template).

> * It allows you to export the project to a given platform. It will allow you to quickly configure the build process and the packagement.

The official template builds for macOS, Windows, Linux and Wasm. iOS and Android are not yet "easy" enough to be included.

> * It associates configuration to assets like SVGs, PNGs and fonts, what are called `.import` files (similiar to `some-svg.svg.import`). A directory can be ignored with `.gdignore`, thus causing the editor to not associate this configuration with these files in that directory recursively.

Bevy asset management doesn't yet have those capabilities, but coming soon!
